Gemfile

```diff
@@ -46,7 +46,7 @@ end
 group :development do
   gem 'rails-dev-tweaks'
   gem 'sprockets' # just so above works
-
+  gem 'thin'
```

Comment:
Vignesh found that adding this gem could help the slow problem when running locally. I feel it really helps too.